.gitattributes

```diff
@@ -1,8 +1,37 @@
-docs/ export-ignore
+/tests export-ignore
+/.tx export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/.scrutinizer.yml export-ignore
+/*.dist export-ignore
+/behat.yml export-ignore
+
+# all client-side source except styles
+/.storybook export-ignore
+/client/src/**/*.js export-ignore
+/client/src/boot export-ignore
+/client/src/bundles export-ignore
+/client/src/components export-ignore
+/client/src/containers export-ignore
+/client/src/font export-ignore
+/client/src/images export-ignore
+/client/src/legacy export-ignore
+/client/src/lib export-ignore
+/client/src/sprites export-ignore
+/client/src/state export-ignore
+/client/src/stories export-ignore
+/client/src/.modernizrrc export-ignore
```

Comment:
I needed to list all `/client/src` sub-directories as `.gitattributes` does not allow for negation selection like in `.gitignore`